Jenkinsfile

```diff
@@ -150,9 +150,10 @@ pipeline {
 									'comma_ver='+swtVersions['comma_ver'], "new_comma_ver=${swtVersions['maj_ver']},${swtVersions['min_ver']},${swtVersions['new_rev']},0" ]) {
 								sh '''
```

Comment:
`+(pattern‐list)` syntax is a bashism (extglob). This won't work in POSIX sh.

We need to switch to bash here and enable extglob:

```diff
-sh '''
+sh '''#!/bin/bash -ex
+      shopt -s extglob
```

Reply:
@sratz : what about 

```
swtNativeLibName='swt-.*-[0-9]+r[0-9]+'
ls -d binaries/org.eclipse.swt.gtk.*/*   | grep -E ${swtNativeLibName}.so     | xargs rm -vf
ls -d binaries/org.eclipse.swt.win32.*/* | grep -E ${swtNativeLibName}.dll    | xargs rm -vf
ls -d binaries/org.eclipse.swt.cocoa.*/* | grep -E ${swtNativeLibName}.jnilib | xargs rm -vf
```

Reply:
thanks for that hint! It's really not simple to understand all these shell scripts in detail 😅

With all this special setup I think it is best, to just go the simple way and use:
```
rm binaries/org.eclipse.swt.gtk.*/libswt-*.so
rm binaries/org.eclipse.swt.win32.*/swt-*.dll
rm binaries/org.eclipse.swt.cocoa.*/libswt-*.jnilib
```
With the current schema this deletes exactly the binaries we build in the SWT pipeline.
All native libraries in the fragments that should not be deleted at this point simply must not start with `swt/libswt`, which is the case for the only current candidate `WebView2Loader.dll`.